cppseminar/presentation/Services/SubmissionService.cs

```diff
@@ -58,6 +58,45 @@ public async Task<IList<Submission>> GetUserSubmissionsAsync(string userEmail)
             }
         }
 
+        public async Task<int> GetNumberOfPages(string userEmail)
+        {
+            userEmail ??= "";
+            userEmail = HttpUtility.UrlEncode(userEmail);
```

Comment:
userEmail = HttpUtility.UrlEncode(userEmail ?? ""); is preffered instead of the line just before this one.